modules/ding_campaign_plus/js/ding_campaign_plus.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @file
+ * Handle campaigns.
+ */
+(function ($) {
+  'use strict';
+
+  /**
+   * Handle trigger type auto-complete callbacks.
+   *
+   * @type {{attach: Drupal.behaviors.ding_campaign_plus_basic_triggers.attach}}
+   */
+  Drupal.behaviors.ding_campaign_plus_node_edit = {
+    attach: function (context, settings) {
+      $('.form-item-ding-campaign-plus-auto-generate-keywords').once(function () {
```

Comment:
This set a 'bar-processed' class when the behaviour is attached - and ensures the behaviour is only attached once.
$('.foo', context).once('bar', function () {
  // do stuff;
});

Reply:
Yes?

Reply:
It does what I wanted?

Reply:
It runs the code once, every time behaviours are attached. If that's what you want, you're good to go ;-)